infra/base-images/base-runner/coverage

```diff
@@ -259,6 +262,11 @@ for fuzz_target in $FUZZ_TARGETS; do
     echo "Running $fuzz_target"
     run_fuzz_target $fuzz_target &
 
+    # Rewrite object if its a FUZZTEST target
```

Comment:
Does this mean that we will measure coverage per binary and not per FUZZ_TEST?

Reply:
Yes, but that's a mistake. I'll fix it up so we get per FUZZ_TEST as well as all FUZZ_TEST combined

Reply:
Fixed!